test/Core.Test/Services/DeviceServiceTests.cs

```diff
@@ -117,12 +116,11 @@ await sutProvider.GetDependency<IDeviceRepository>()
             .Received(3)
             .UpsertAsync(Arg.Any<Device>());
 
-        // TODO: .NET 8: Use nameof for parameter name.
-        static void SetupOldTrust(Device device, [CallerArgumentExpression("device")] string expression = null)
+        static void SetupOldTrust(Device device)
```

Comment:
This is my fault for the bad comment, `[CallerArgumentExpression]` gets the string expression from what the caller has. So, calling `SetupOldTrust(deviceOne)` compiles to: `SetupOldTrust(deviceOne, "deviceOne")`. My mention of using `nameof` was for the string argument in `[CallerArgumentExpression]` because you can now reference parameter names using nameof inside of argument attributes so my intended upgrade was to change it to `[CallerArgumentExpression(nameof(device)]`.